public/app/plugins/datasource/influxdb/components/InfluxCheatSheet.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const CHEAT_SHEET_ITEMS = [
+  {
+    title: 'Getting started',
+    label: 'Start by selecting a measurement and field from the dropdown above.',
```

Comment:
... and then hit "Run Query".

Also would be good to describe the tag chooser to narrow down your search.

Reply:
👍